### DISCUSSION TITLE
Tauri's file dialog doesn't support creating new folder, is this a missing function?

### DISCUSSION THREAD

**Discussion by shi-yan** [tauri-apps/tauri#6491] [Q&A]
I'm on mac, I tried the file dialog (directory: true) for my users to pick a directory for their files. 

If there is no suitable directory, I hope they can directly create a new folder within the dialog.

But tauri's file dialog doesn't seem to allow that (only tested on mac). I'm not familiar with Mac's native API, but I know, for example, Qt's file dialog can let users create folders. 

So I'm wondering if this is something missed.

**Comment by FabianLars** [comment]
This will be tracked here: <https://github.com/tauri-apps/tauri/issues/6435>. I think we just never noticed that until now tbh.
The issue also includes a link to the obj-c api i found which hopefully works for us.
